department-of-veterans-affairs/caseflow
# Adds linkage between Reader and Queue

Connects https://github.com/department-of-veterans-affairs/caseflow/issues/4368

![reader_link](https://user-images.githubusercontent.com/3781835/36114676-ecd2408c-0fed-11e8-8af6-8116e93da607.gif)

I initially made an action and a reducer, but removed them since we really only need an initial state value for this — we should never need to change it.

## Testing
- `FeatureToggle.enable!(:queue_welcome_gate)`
- Click from Queue list view into Reader. Reader nav bar does not contain "Claims Folder" breadcrumb.
- Click "Caseflow Reader" in the Reader nav bar. It takes you to the documents list page and not the reader welcome gate.
- Click from the Queue task list view into Reader. A "Back to Your Queue" link is displayed that leads back to `/queue`
- Click from the Queue task details view into Reader. A "Back to Draft Decision - Veteran Name (VBMSID)" link is displayed that leads back to the task details view.
